integreat_cms/static/src/index.ts

```diff
@@ -76,6 +76,8 @@ import "./js/unsaved-warning";
 
 import "./js/pois/poi-actions.ts";
 
+import "./js/dashboard.ts";
```

Comment:
```suggestion
import "./js/menu.ts";
```
Dashboard doesn't really make sense when the menu is visible on every page, not only the dashboard.

Reply:
Yes, I agree :) Thank you!